test/test_emoji_log.py

```diff
@@ -9,6 +9,9 @@ def test_emoji_log(uqcsbot: MockUQCSBot):
     Test !emoji_log for a member adding or removing emoji.
     """
     events = [
+        generate_event_object(MESSAGE_TYPE_CHANNEL_CREATED,

```

Comment:
Why are we adding uqcs-meta to the emoji tests?

Reply:
The next event creates an #emoji-request channel, which triggers the `channel_log` function. Without a #uqcs-meta channel to post in, the `channel_log` function errors.

Reply:
Add a comment pls

Reply:
Added.